kolibri/plugins/facility/assets/src/views/DataPage/SyncInterface/index.vue

```diff
@@ -59,16 +59,15 @@
                 >
                   <template #options>
                     <CoreMenuOption
-                      v-if="theFacility.dataset.registered"
```

Comment:
We should not be messing with the existence registration code, and should instead be adding an additional `CoreMenuOption` for the manage sync.